VocaDb.ResXFileCodeGenerator.Tests/GithubIssues/FeatureRequests/MethodGeneratorTests.cs

```diff
@@ -0,0 +1,144 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Xunit;
+
+namespace VocaDb.ResXFileCodeGenerator.Tests.GithubIssues.FeatureRequests;
+public class MethodGeneratorTests
+{
+	[Fact]
+	public void Generate_StringBuilder_Name_NotValidIdentifier()
+	{
+		var text = @"<?xml version=""1.0"" encoding=""utf-8""?>
```

Comment:
Can you please use [raw string literals](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/raw-string)?